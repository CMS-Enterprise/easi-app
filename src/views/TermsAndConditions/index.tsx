import React from 'react';
import { useTranslation } from 'react-i18next';

import Header from 'components/Header';
import MainContent from 'components/MainContent';

const TermsAndConditions = () => {
  const { t } = useTranslation('termsAndConditions');
  return (
    <div>
      <Header />
      <MainContent className="grid-container line-height-body-5">
        <h1 className="font-heading-xl">{t('heading')}</h1>
        <h2>{t('subheading')}</h2>
        <p className="margin-y-4">{t('unauthorizedUse')}</p>
        <p className="margin-y-4">{t('socialMediaUse')}</p>
        <p className="margin-y-4">{t('consent')}</p>
        <p className="margin-y-4">{t('infoStorage')}</p>
      </MainContent>
    </div>
  );
};

export default TermsAndConditions;
