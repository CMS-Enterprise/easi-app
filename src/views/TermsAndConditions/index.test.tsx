import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { render } from '@testing-library/react';

import TermsAndConditions from './index';

vi.mock('@okta/okta-react', () => ({
  useOktaAuth: () => {
    return {
      authState: {
        isAuthenticated: true
      },
      oktaAuth: {
        getUser: async () => {},
        logout: async () => {}
      }
    };
  }
}));

describe('The Terms & Conditions page', () => {
  it('matches the snapshot', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <TermsAndConditions />
      </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
