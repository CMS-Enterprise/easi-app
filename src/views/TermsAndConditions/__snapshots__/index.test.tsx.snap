// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Terms & Conditions page matches the snapshot 1`] = `
<div
  className="easi-page-wrapper"
>
  <header
    className="usa-header easi-header"
    role="banner"
  >
    <section
      aria-label="Official government website"
      className="usa-banner"
    >
      <div
        className="usa-accordion"
      >
        <header
          className="usa-banner__header"
        >
          <div
            className="usa-banner__inner"
          >
            <div
              className="grid-col-auto"
            >
              <img
                alt="U.S. flag"
                className="usa-banner__header-flag"
                src="us_flag_small.png"
              />
            </div>
            <div
              className="grid-col-fill tablet:grid-col-auto"
            >
              <p
                className="usa-banner__header-text"
              >
                An official website of the United States government
              </p>
              <p
                aria-hidden="true"
                className="usa-banner__header-action"
              >
                Here’s how you know
              </p>
            </div>
            <button
              aria-controls="gov-banner"
              aria-expanded={false}
              className="usa-accordion__button usa-banner__button"
              onClick={[Function]}
              type="button"
            >
              <span
                className="usa-banner__button-text"
              >
                Here’s how you know
              </span>
            </button>
          </div>
        </header>
        <div
          className="usa-banner__content usa-accordion__content"
          hidden={true}
          id="gov-banner"
        >
          <div
            className="grid-row grid-gap-lg"
          >
            <div
              className="usa-banner__guidance tablet:grid-col-6"
            >
              <img
                alt="Dot gov"
                className="usa-banner__icon usa-media-block__img"
                src="icon-dot-gov.svg"
              />
              <div
                className="usa-media-block__body"
              >
                <p>
                  <strong>
                    Official websites use .gov
                  </strong>
                  <br />
                  A 
                  <strong>
                    .gov
                  </strong>
                   website belongs to an official government organization in the United States.
                </p>
              </div>
            </div>
            <div
              className="usa-banner__guidance tablet:grid-col-6"
            >
              <img
                alt="Https"
                className="usa-banner__icon usa-media-block__img"
                src="icon-https.svg"
              />
              <div
                className="usa-media-block__body"
              >
                <p>
                  <strong>
                    Secure .gov websites use HTTPS
                  </strong>
                  <br />
                  A 
                  <strong>
                    lock
                  </strong>
                   (
                  <span
                    className="icon-lock"
                  >
                    <svg
                      aria-labelledby="banner-lock-title banner-lock-description"
                      className="usa-banner__lock-image"
                      height="64"
                      role="img"
                      viewBox="0 0 52 64"
                      width="52"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <title
                        id="banner-lock-title"
                      >
                        Lock
                      </title>
                      <desc
                        id="banner-lock-description"
                      >
                        A locked padlock
                      </desc>
                      <path
                        d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"
                        fill="#000000"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                  ) or 
                  <strong>
                    https://
                  </strong>
                   means you’ve safely connected to the .gov website. Share sensitive information only on official, secure websites.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
    <div
      className="grid-container easi-header__basic"
    >
      <div
        className="usa-logo site-logo"
        id="logo"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <em
            aria-label="EASi home"
            className="usa-logo__text"
          >
            EASi
          </em>
        </a>
      </div>
      <button
        className="usa-menu-btn"
        type="button"
      >
        <span
          className="fa fa-bars"
        />
      </button>
      <div
        className="navbar--container"
      >
        <div
          className="easi-header__dropdown-wrapper"
        >
          <button
            aria-controls="Header-UserActionsList"
            aria-expanded={false}
            aria-label="Expand User Menu"
            className="easi-header__username"
            onClick={[Function]}
            type="button"
          >
            
            <i
              className="fa fa-angle-down easi-header__caret"
            />
          </button>
        </div>
      </div>
    </div>
    <div
      className="grid-container easi-header--desktop "
    />
    <div
      className="usa-nav sidenav-mobile"
    >
      <button
        aria-label="Close"
        className="usa-nav__close"
        type="button"
      >
        <span
          className="fa fa-close"
        />
      </button>
      <div
        className="usa-nav__inner"
      >
        <button
          className="easi-header__nav-link"
          onClick={[Function]}
          type="button"
        >
          Sign Out
        </button>
      </div>
    </div>
  </header>
  <main
    className="easi-main-content grid-container line-height-body-5"
    id="main-content"
    role="main"
    tabIndex={-1}
  >
    <h1
      className="easi-h1"
      tabIndex={-1}
    >
      Terms & Conditions
    </h1>
    <h2>
      This system is provided for Government authorized use only.
    </h2>
    <p
      className="margin-y-4"
    >
      Unauthorized or improper use of this system is prohibited and may result in disciplinary action and/or civil and criminal penalties.
    </p>
    <p
      className="margin-y-4"
    >
      Personal use of social media and networking sites on this system is limited as to not interfere with official work duties and is subject to monitoring.
    </p>
    <p
      className="margin-y-4"
    >
      By using this system, you understand and consent to the following: The Government may monitor, record, and audit your system usage, including usage of personal devices and email systems for official reasonable expectation of privacy regarding any communication or data transiting or stored on this system. At any time, and for any lawful Government purpose, the government may monitor, intercept, and search and seize any communication or data transiting or stored on this system.
    </p>
    <p
      className="margin-y-4"
    >
      Any communication or data transiting or stored on this system may be disclosed or used for any lawful Government purpose.
    </p>
  </main>
  <footer
    className="usa-footer usa-footer--slim"
  >
    <div
      className="usa-footer__primary-section"
    >
      <div
        className="usa-footer__primary-container grid-row"
      >
        <div
          className="mobile-lg:grid-col-8"
        >
          <nav
            className="usa-footer__nav"
          >
            <ul
              className="grid-row grid-gap"
            >
              <li
                className="desktop:grid-col-auto usa-footer__primary-content mobile-lg:grid-col-6"
              >
                <a
                  className="usa-footer__primary-link"
                  href="/privacy-policy"
                >
                  Privacy Policy
                </a>
              </li>
              <li
                className="desktop:grid-col-auto usa-footer__primary-content mobile-lg:grid-col-6"
              >
                <a
                  className="usa-footer__primary-link"
                  href="/cookies"
                >
                  Cookies
                </a>
              </li>
              <li
                className="desktop:grid-col-auto usa-footer__primary-content mobile-lg:grid-col-6"
              >
                <a
                  className="usa-footer__primary-link"
                  href="/terms-and-conditions"
                >
                  Terms and Conditions
                </a>
              </li>
              <li
                className="desktop:grid-col-auto usa-footer__primary-content mobile-lg:grid-col-6"
              >
                <a
                  className="usa-footer__primary-link"
                  href="/accessibility-statement"
                >
                  Accessibility Statement
                </a>
              </li>
            </ul>
          </nav>
        </div>
      </div>
    </div>
    <div
      className="usa-footer__secondary-section"
    >
      <div
        className="grid-container"
      >
        <div>
          <div
            className="easi-footer__secondary-logo-wrap"
          >
            <img
              alt="CMS.gov Logo"
              src="cmsGovLogo.png"
            />
            <img
              alt="Department of Health and Human Services USA"
              className="margin-left-1"
              src="hhsLogo.png"
            />
          </div>
          <span>
            A federal government website managed and paid for by the U.S. Centers for Medicare & Medicaid Services. 7500 Security Boulevard, Baltimore, MD 21244
          </span>
        </div>
      </div>
    </div>
  </footer>
</div>
`;
